### ISSUE TITLE
Create test app using ruby-kafka

### ISSUE THREAD

**Issue by tannalynn** [newrelic/newrelic-ruby-agent#2760]
Using the ruby-kafka gem, create a test app that will talk to the aws kafka service.

**Comment by mensfeld** [comment]
hey, friendly notice, that ruby-kafka is fully deprecated: https://github.com/zendesk/ruby-kafka

as stated there is is replaced by https://github.com/karafka/rdkafka-ruby

**Comment by tannalynn** [comment]
Hello! Thank you. 

I did see that at when i started looking into the kafka gems, but after looking into it internally, we saw that we actually still have a decent amount of customers that are still using the ruby-kafka gem despite that. So to support those customers, we decided to add instrumentation for ruby-kafka as well, even though it is deprecated. 

We don't actually use kafka for the agent itself, this issue is only to create a test app that simulates a customer using this gem with the ruby agent installed. Also, instrumentation added for ruby-kafka will only be installed on applications using the ruby-kafka gem, so customers using rdkafka will be unaffected by the ruby-kafka instrumentation.

Hopefully that makes sense, let me know if you have any other questions.
